platform/darwin/src/MGLNetworkConfiguration.h

```diff
@@ -6,18 +6,13 @@ NS_ASSUME_NONNULL_BEGIN
  The MGLNetworkConfiguration object provides a global way to set a base API URL for
  retrieval of map data, styles, and other resources.
 
- Currently, MGLNetworkConfiguration is private API in code but is able to be used
- by any applications via the `MGLMapboxAPIBaseURL` dictionary key in the
- application's `Info.plist`.
+ Currently, MGLNetworkConfiguration is a private API.
```

Comment:
- Was this documentation no longer valid? Where is this information captured now?
- If `MGLNetworkConfiguration` is private, why is it not named `MGLNetworkConfiguration_Private.h`?

Reply:
I'm taking care of this in https://github.com/mapbox/mapbox-gl-native/pull/13886 this is becoming a public class